pkg/rpm/rpmbuild/SOURCES/unit.service

```diff
@@ -15,12 +15,13 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-Type=simple
+Type=forking
 Environment="UNITD_OPTIONS=--log /var/log/unit/unit.log --pid /var/run/unit/unit.pid"
-ExecStart=/usr/sbin/unitd $UNITD_OPTIONS --no-daemon
+ExecStart=/usr/sbin/unitd $UNITD_OPTIONS
 ExecReload=
 RuntimeDirectory=unit
 RuntimeDirectoryMode=0755
+PIDFile=/var/run/unit/unit.pid
```

Comment:
Can technically just be `/run/unit/unit.pid`

`/var/run` has been a symlink to `/run` for I don't know how many years... but no biggie...

But yeah, looks good.

Reply:
Oh yeah, let's make it contemporary.  I've checked Amazon Linux 2 and CentOS 7, and they already have a symlink, so we should just fix that (which I just did).